notebooks/facloc-disperse-real-world.ipynb

```diff
@@ -928,7 +976,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 130,
+   "execution_count": 367,
```

Comment:
It looks like something strange may be going on here. We are defining $p=6$, but there are 9 facilities in the solution set. It appears that the model *is* taking the predefined facilities into consideration, but taking 6 more candidate slots onto that...

Reply:
I'm curious what the LP file looks like. Can you print that out and post it here (`.problem`)?

Reply:
I'm pretty certain what you're seeing is from me toying with the plotting function and getting undesirable results.
![Screen Shot 2022-08-17 at 5 51 26 PM](https://user-images.githubusercontent.com/39073850/185249802-894d0603-112a-4961-aa87-4c797b446772.png)

Reply:
hmmm. I see your results of 6 selected facilities in cell 38 in the real world notebook after you reran it. So you seem to be right that the actual model is being formulated and solved correctly. And it does seem to be something that is happening funkily in the indexing somewhere between dataframe index, predefined tag, and decision variable. We need to track down the trouble and make 100% sure that is the case. Or find out what is actually going on.

@gegen07 Do you have time to help troubleshoot this?

Reply:
I switched things up a bit to symbolize the predefined with a red outline. To do this I filtered the dataframe, but that shifts all the indexes, so not great. I think I can get it sorted tomorrow.